cohort-engine/src/test/java/com/ibm/cohort/engine/measure/MeasureEvaluatorTest.java

```diff
@@ -644,4 +657,52 @@ public void in_populations_supplemental_data() throws Exception {
 		
 		// See MeasureSupplementalDataEvaluationTest for more details on what is returned here
 	}
+
+	@Test
+	public void measure_report_generated___named_parameters_on_measure_report() throws Exception {
+		CapabilityStatement metadata = getCapabilityStatement();
+		mockFhirResourceRetrieval("/metadata", metadata);
+
+		Patient patient = getPatient("123", AdministrativeGender.MALE, "1970-10-10");
+		mockFhirResourceRetrieval(patient);
+
+		Library library = mockLibraryRetrieval("TestDummyPopulations", DEFAULT_VERSION, "cql/fhir-measure/test-dummy-populations.xml",
+											   LibraryFormat.MIME_TYPE_APPLICATION_ELM_XML);
+
+		Measure measure = getCohortMeasure("CohortMeasureName", library, INITIAL_POPULATION);
+		mockFhirResourceRetrieval(measure);
+
+		Map<String, Parameter> parameterMap = new HashMap<>();
+		parameterMap.put("integerParam", new IntegerParameter(1));
+		parameterMap.put("decimalParam", new DecimalParameter("1.0"));
+		parameterMap.put("stringParam", new StringParameter("1"));
+		parameterMap.put("booleanParam", new BooleanParameter(false));
+		parameterMap.put("datetimeParam", new DatetimeParameter("2020-01-01"));
```

Comment:
It would be good to add a test for a DatetimeParameter that includes the timezone information to demonstrate and verify the behavior when the user provides a non-UTC timezone.

Reply:
I had considered this case, but it sort of felt like more of a test that was specific to DatetimeParameter. I'm fine still  adding a test to cover this.

Reply:
Added a test.